iOS/Flows/New Conversation/ConversationThreadViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  ConversationViewController.swift
+//  Benji
+//
+//  Created by Benji Dodgson on 12/27/18.
+//  Copyright © 2018 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import Parse
+import Combine
+import StreamChat
+
+class ConversationThreadViewController: BlurredViewController, CollectionViewInputHandler {
```

Comment:
As mentioned above, this is renamed. It's pretty much the same as before except that it takes in a message, not a conversation, and all the data logic is centered around threads not conversations.